forge-gui/res/cardsfolder/upcoming/effluence_devourer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Effluence Devourer
+ManaCost:B R G
+Types:Creature Crocodile
+PT:4/3
+K:Blitz:B R G
+T:Mode$ Sacrificed | ValidCard$ Creature | ValidPlayer$ You | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice CARDNAME or another creature, it perpetually gains “{2}, Exile this card from your graveyard: Create an X/X green Ooze creature token, where X is this card's power. Activate only as a sorcery.”
```

Comment:
This needs to work even if Effluence Devourer somehow isn't a creature... currently it won't